README.md

```diff
@@ -41,6 +41,10 @@ To make changes to the rhacs dashboards:
 * Run `make generate` to generate the corresponding resources for the Grafana operator.
 
 To make changes to Kubernetes mixin resources:
+
+First, make sure you have `go-jsonnet` and `jsonnet-bundler` installed, and run `(cd resources/mixins/kubernetes && jb install github.com/kubernetes-monitoring/kubernetes-mixin)` to set things up.
```

Comment:
I think is idempotent. So we can just add it to the `make generate`/`make update` commands instead of documenting them.

Reply:
Great idea, it seems to be very fast on re-run. PTAL